### PR TITLE
enhance: add meaningful field in log/error msg

### PR DESCRIPTION
**Pull request by houstar**
Signed-off-by: Leno Hou <lenohou@gmail.com>

### Ⅰ. Describe what this PR did
Drop by the gometalinter to check the code  :-)

### Ⅱ. Does this pull request fix one issue?

N/A

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
N/A since it's just fix the code-style

### Ⅳ. Describe how to verify it
gometalinter ./... to reduce some gofmt issue

### CODE REVIEW

File: cri/stream/portforward/httpstream.go [hunk 1]
```diff
@@ -74,8 +74,8 @@ func handleHTTPStreams(ctx context.Context, w http.ResponseWriter, req *http.Req
 		streamChan:            streamChan,
 		streamPairs:           collect.NewSafeMap(),
 		streamCreationTimeout: streamCreationTimeout,
-		pod:       podName,
-		forwarder: portForwarder,
+		pod:                   podName,
```
Comment (allencloud) [inline cri/stream/portforward/httpstream.go]: I am just curious about why the integrated CI has not detected this part? @houstar @fuweid @Starnop 😕

**Comment by allencloud** [comment]
I am just curious about why the integrated CI has not detected this part?

And the circleCI has been triggered, it is strange. Have you synced with the latest master branch? @houstar

**Comment by houstar** [comment]
@allencloud 
YES. this is the latest master branch with commit id 4cce334b130f6e9be8873bacb43e6274d1423b12

File: cri/stream/portforward/httpstream.go (hunk 1 above)
Comment (fuweid) [reply cri/stream/portforward/httpstream.go]: I think it is bug in the gometerlinter. The megachecker process was killed by OOM, which might cause the following the check doesn't work. @allencloud

**Comment by houstar** [comment]
@fuweid  @allencloud Maybe we can enable gometalinter check on our pouch-robot since it's running on our infrastructure with large memory. WDYT?

**Comment by fuweid** [comment]
@houstar just curiosity, why is there no circleci job? 🤔

**Comment by HusterWan** [comment]
please rebase your PR to fix the unittest failed, thanks

File: ctrd/container.go [hunk 2]
```diff
@@ -603,7 +603,7 @@ func (c *Client) createTask(ctx context.Context, id, checkpointDir string, conta
 
 	statusCh, err := task.Wait(context.TODO())
 	if err != nil {
-		return pack, errors.Wrapf(err, "failed to wait task in container", id)
+		return pack, errors.Wrapf(err, "failed to wait task in container(%s)", id)
```
Comment (allencloud) [inline ctrd/container.go]: good catch!

**Comment by allencloud** [comment]
Thanks for your work. While I think what you changed in log and error message mean a lot. 
However how about just reverting the gofmt part of thing. Thanks a lot. @houstar

**Comment by fuweid** [comment]
The indent issue is caused by the go1.11.x version. Since we are still using the 110.4, I think we should keep it unchanged. @houstar

**Pull request merged**